cmd/bpf2go/compile.go

```diff
@@ -39,6 +39,7 @@ func compile(args compileArgs) error {
 	cmd.Args = append(cmd.Args,
 		"-c", args.source,
 		"-o", args.dest,
+		"-O2",
```

Comment:
I think by putting it here it's not possible to specify e.g. `-O3` anymor. This later `-O2` will override the earlier flag. Can you put the "-O2" at the start of cmd.Args instead?

Reply:
Right, I'll fix it

Reply:
v2 pushed